tests/unit/fem/test_quadf_coef.cpp

```diff
@@ -156,6 +157,73 @@ TEST_CASE("Quadrature Function Coefficients",
       REQUIRE(output.Norml2() < tol);
    }
 
+   SECTION("Write and Read Scalar Quadraturefunction")
```

Comment:
So the changes made to this unit test really don't belong in this file. They would be better suited in the /tests/unit/fem/test_datacollection.cpp file where the other data collection are. So, I would suggest reverting changes made here and moving them over to the other file.

Reply:
Indeed don't know why I did not consider that. Will move, and change the test to better fit the datacollection unittest.